check_isbn.py

```diff
@@ -0,0 +1,5 @@
+import os
+from imports import *
+
+def check_isbn(args):
+    return db.session.query(Book).filter(Book.isbn== args)
```

Comment:
This implimentation will throw error because, session of Database(db) is not imported into this file, Try running the application with the updated changees, You will get the errors and add imports accordingly